crates/autopilot/src/run_loop.rs

```diff
@@ -147,6 +147,42 @@ impl RunLoop {
                 }
             };
 
+            {
+                // Also calculate the score from the solution itself
```

Comment:
This is a temporary code that will be replaced when we switch to using autopilot calculated score. So did not spend too much time polishing it. It's used to detect differences in driver/autopilot score calculations.